PythonPonies/bangazon-api
# creating initial django app

# Description
Adding the initial Django app we'll need to use the Django REST Framework. I also added a .gitignore file.

## Number of Fixes
I did 3 things.

1. Added a `bangazon` django app
1. The default app `bangazon` is named the same thing and within the root directory
1. Added a gitignore file

## Related Ticket(s)
#1 

## Problem to Solve
We wanted a quick way to build a Restful API.

## Proposed Changes
None

## Expected Behavior
We'll be able to build out an API with the Django REST Framework.

## Steps to Test Solution

1. Git fetch this pull request.
2. Make sure Django files are intact.
3. You should be able to run the server without error.

## Testing

No testing for this test.

## Documentation
No documentation needed for this step.

## Deploy Notes
None
